game-app/game-core/src/test/resources/required-op-codes.csv

```diff
@@ -50,7 +50,6 @@
 3,games.strategy.engine.framework.startup.mc.IServerStartupRemote,changeToLatestAutosave,games.strategy.engine.framework.HeadlessAutoSaveType
 8,games.strategy.engine.framework.startup.mc.IServerStartupRemote,getIsServerHeadless
 6,games.strategy.engine.player.Player,getName
-9,games.strategy.engine.player.Player,initialize,games.strategy.engine.player.IPlayerBridge,games.strategy.engine.data.GamePlayer
```

Comment:
As I mentioned already this method can't be called over the network unless the implementation is serializable which is not the case.